CRITICAL - 2021-03-17 09:51:21 --> Undefined array key "foto"
#0 C:\xampp\htdocs\cs\app\Controllers\Auth.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 58)
#1 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Auth->authenticate()
#2 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-03-17 14:25:32 --> Undefined array key "foto"
#0 C:\xampp\htdocs\cs\app\Controllers\Auth.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 58)
#1 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Auth->authenticate()
#2 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-03-17 18:16:04 --> Undefined array key "foto"
#0 C:\xampp\htdocs\cs\app\Controllers\Auth.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 58)
#1 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Auth->authenticate()
#2 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 {main}
ERROR - 2021-03-17 18:32:14 --> mysqli_sql_exception: Unknown column 'u.foto' in 'field list' in C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php(314): mysqli->query('SELECT CONCAT(u...')
#1 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('SELECT CONCAT(u...')
#2 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT CONCAT(u...')
#3 C:\xampp\htdocs\cs\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT CONCAT(u...', Array, false)
#4 C:\xampp\htdocs\cs\app\Models\PerfilModel.php(52): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\cs\app\Controllers\Perfil.php(26): App\Models\PerfilModel->datos_usuario_perfil('2')
#6 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Perfil->datos_usuario()
#7 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Perfil))
#8 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-03-17 18:32:14 --> Unknown column 'u.foto' in 'field list'
#0 C:\xampp\htdocs\cs\system\Database\MySQLi\Connection.php(314): mysqli->query('SELECT CONCAT(u...')
#1 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('SELECT CONCAT(u...')
#2 C:\xampp\htdocs\cs\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT CONCAT(u...')
#3 C:\xampp\htdocs\cs\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT CONCAT(u...', Array, false)
#4 C:\xampp\htdocs\cs\app\Models\PerfilModel.php(52): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\cs\app\Controllers\Perfil.php(26): App\Models\PerfilModel->datos_usuario_perfil('2')
#6 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Perfil->datos_usuario()
#7 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Perfil))
#8 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-03-17 20:57:46 --> Undefined array key "foto"
#0 C:\xampp\htdocs\cs\app\Controllers\Auth.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 58)
#1 C:\xampp\htdocs\cs\system\CodeIgniter.php(936): App\Controllers\Auth->authenticate()
#2 C:\xampp\htdocs\cs\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\cs\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\cs\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 {main}
